src/config/constants.ts

```diff
@@ -135,3 +135,8 @@ export const TEAM_MAX_SIZE = 'Max 4 Hackers';
 export const SPONSOR_COMPANY_LABEL = 'Company name';
 export const SPONSOR_CONTRACT_URL_LABEL = 'Contract URL';
 export const SPONSOR_NOMINEE_LABEL = 'Nominees';
+
+// Settings management page
+export const SETTINGS_OPEN_TIME_LABEL = 'Applications Open At';
+export const SETTINGS_CLOSE_TIME_LABEL = 'Applications Close At';
+export const SETTINGS_CONFIRM_TIME_LABEL = 'Hacker Confirmations Close At';
```

Comment:
nit: sentence case if it's more than just a label of the field (i.e. First Name vs What's your first name? need better docs on ux copy tho)

```suggestion
export const SETTINGS_OPEN_TIME_LABEL = 'Applications open at:';
export const SETTINGS_CLOSE_TIME_LABEL = 'Applications close at:';
export const SETTINGS_CONFIRM_TIME_LABEL = 'Hacker confirmations close at:';
```